Hi.

I have a better solution to the mce reinit issue. This patch also
removes the warning that the previous iteration created.

Regards,

Nigel

 suspend2.h |    1 +
 1 files changed, 1 insertion(+)
diff -ruNp 9010-mce-config.patch-old/include/asm-i386/suspend2.h 9010-mce-config.patch-new/include/asm-i386/suspend2.h
--- 9010-mce-config.patch-old/include/asm-i386/suspend2.h	2005-12-28 09:59:00.000000000 +1000
+++ 9010-mce-config.patch-new/include/asm-i386/suspend2.h	2005-12-28 07:03:07.000000000 +1000
@@ -10,6 +10,7 @@
 #include <asm/i387.h>
 #include <asm/tlbflush.h>
 #include <asm/desc.h>
+#include <asm/processor.h>
 
 /* image of the saved processor states */
 struct suspend2_saved_context {



_______________________________________________
Suspend2-devel mailing list
Suspend2-devel@lists.suspend2.net
http://lists.suspend2.net/mailman/listinfo/suspend2-devel

